README.md

```diff
@@ -208,17 +171,12 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Authors**
+Outkarm Johnson
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [Outkarm](https://github.com/Outkarm)
+- GitHub: [Outkarm](https://github.com/Outkarm)
+- LinkedIn: [John Kpordje](https://www.linkedin.com/in/john-kpordje-866749241/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224508741-93a66af4-d9ff-43d3-8e5c-379db8336779.png)

**[OPTIONAL]**

- [ ] Kudos on fixing the Issues page link 👍 It would be nice if you fix your MIT link 😊